spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -110,6 +110,7 @@
 import org.springframework.core.convert.ConversionService;
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.core.convert.support.DefaultConversionService;
+import org.springframework.lang.NonNull;
```

Comment:
This cannot be backported to `1.7.x`  - Spring `5.0` matter.
Will be replaced with the `@Payload`.